ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -60,7 +76,6 @@ export function createInitialTimeSeriesChartOptions(): TimeSeriesChartOptions {
         },
       },
     ],
-    legend: DEFAULT_LEGEND,
```

Comment:
Why did this line get removed? Guessing I missed something.

Reply:
Oh whoops I meant to call this out or I probably should have done it in a separate PR, this was an oversight I made when responding to feedback on the original legend options PR [here](https://github.com/perses/perses/pull/660#discussion_r998787231). 

Since I got rid of the `legend.show` property that means if legend is populated then it will be on by default, but we want legend to be off by default